src/fides/lib/models/fides_user.py

```diff
@@ -79,6 +81,7 @@ def create(cls, db: Session, data: dict[str, Any]) -> FidesUser:
                 "first_name": data.get("first_name"),
                 "last_name": data.get("last_name"),
             },
+            check_name=check_name,
```

Comment:
yeah, at least i _should_, though i think python is permissive enough where this doesn't actually have an impact at runtime. i'm surprised these weren't caught by the type checker or linter, though. it was caught for this class, but not others what you're pointing out.

let me go through a bit more thoroughly with text search rather than relying on the static analysis tools to help me out :)

Reply:
those seem to be the only other two files that were missed, as far as i can tell (2 classes `Rule` and `RuleTarget` within `Policy.py`). 

i'll make those updates, thanks for catching this!